Flutter/lib/CreateSettle.dart

```diff
@@ -67,36 +60,38 @@ class _CreateSettle extends State<CreateSettle> {
 
   Future<void> showPopup() async {
     await animated_dialog_box.showScaleAlertBox(
-        title: Center(
-            child: Text(AppLocalizations.of(context).translate("getcode"))),
+        title: Center(child: Text(getText(context, "getcode"))),
         context: context,
-        firstButton: MaterialButton(
-          shape: RoundedRectangleBorder(
-            borderRadius: BorderRadius.circular(40),
-          ),
-          color: Colors.blue,
-          child: Text(
-            AppLocalizations.of(context).translate("golobby"),
-            style: TextStyle(color: Colors.white),
-          ),
-          onPressed: () {
-            Navigator.push( // Go to the lobby
+        firstButton: AppTheme.rawButton(context, "golobby", () {
+          if (gotCode) {
+            return Navigator.push(
               context,
               MaterialPageRoute(
-                builder: (context) => LobbyScreen(settle, hostName, true)
-              ),
+                  builder: (context) =>
+                      LobbyScreen(settle, hostName, true, widget.themeChange)),
             );
-          },
-        ),
-        secondButton: MaterialButton(
+          } else {
+            return null;
+          }
+        }),
+        secondButton: RaisedButton(
+          elevation: 4,
           shape: RoundedRectangleBorder(
             borderRadius: BorderRadius.circular(40),
           ),
-          color: Colors.white,
-          child: Text(AppLocalizations.of(context).translate("sharecopy")),
+          color: themeChange.darkTheme ? Color(0xff5B5B5B) : Colors.white,
```

Comment:
Can't this be placed in AppTheme as well? We're going to be writing this ternary in a whole lot of places. Something like `AppTheme.color` makes more sense and is much easier to modify later if we choose to

Reply:
Resolved in commit `app-style 0181911`